Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.yml

```diff
@@ -0,0 +1,134 @@
+category: Vulnerability Management
+commonfields:
+  id: CofenseFeed
+  version: -1
+configuration:
+- defaultvalue: https://threathq.com
+  display: URL
+  name: url
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: false
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- defaultvalue: F - Reliability cannot be judged
+  display: Reliability
+  name: feedInstanceReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- name: expiration
+  required: false
+  type: 17
+- defaultvalue: indicatorType
+  hidden: true
+  name: expirationPolicy
+  required: false
+  type: 0
+- hidden: true
+  name: expirationInterval
+  required: false
+  type: 1
+- additionalinfo: To fetch malware and phishing indicators select "all".
+  defaultvalue: all
+  display: The threat type for which to fetch indicators.
+  name: threat_type
+  options:
+  - all
+  - malware
+  - phish
+  required: false
+  type: 15
+- defaultvalue: 3 days
+  display: First fetch time range (<number> <time unit>, e.g., 1 hour, 30 minutes)
+  hidden: false
+  name: fetch_time
+  required: false
+  type: 0
+- display: Bypass exclusion list
+  name: bypassExclusionList
```

Comment:
Missing the additionalinfo field for this parameter. You can go to demo7 and create a new integration that fetches indicators and it'll auto-populate the parameters with all the ones that are expected from server and then you can check that you don't have any other parameters missing and/or parameter fields missing and/or parameter field values that have been updated since you worked on the integration for now.

Reply:
Isn't it should be generated by the server?